FabFos.py

```diff
@@ -215,6 +215,17 @@ def format(self, record):
 
 #################################### Classes end ################################################
 
+def bam2fastq(input_file, sample_id, output_dir, executables):
+    fastq_extract = [executables["bam2fastq"], "-o",
+                     output_dir + os.sep + sample_id + input_file + ".fasta", "--no-aligned", input_file]
+    fastq_extract += ["1>", output_dir + os.sep + "bam2fastq.stdout"]
+    fastq_extract += ["2>", output_dir + os.sep + "bam2fastq.stderr"]
+    p_bam2fastq = subprocess.Popen(' '.join(fastq_extract), shell=True, preexec_fn=os.setsid)
```

Comment:
Setting the `shell` parameter to `True` is a security issue and though this probably won't be critical to FabFos, it is frowned upon. Instead, I suggest removing the `' '.join()` call and setting `shell` to False.